elements/lisk-p2p/src/utils/misc.ts

```diff
@@ -121,7 +121,7 @@ export const getNetgroup = (address: string, secret: number): number => {
 };
 
 // TODO: Remove the usage of height for choosing among peers having same ip, instead use productivity and reputation
-export const getUniquePeersbyIp = (
+export const getBestPeersbyHeight = (
```

Comment:
Keep the old name `getUniquePeersbyIp ` as we are not going to rely on height in future